### PR TITLE
LPS-129701 Migrate selection modals in site modules

### PR DESCRIPTION
**Pull request by kresimir-coko**
Part 1 fix of https://issues.liferay.com/browse/LPS-129701.

Here are some of the removals I managed to get done. There are a couple leftover cases that I'll try and fix tomorrow, one of the leftover (not commited) migrations would have to wait for https://github.com/pei-jung/liferay-portal/pull/824 so I just sent this as a part 1 of the task.

We (@victorg1991 and I) also fixed a bug that came up with the Management Toolbar migration via https://github.com/liferay-echo/liferay-portal/commit/f8607cd53a6af4a2997c28fcb594838dc60402c3. 

I've manually tested these and they work, but I can no longer recall the steps to test them individually so I can't write it up for you, sorry.

### CODE REVIEW

**Comment by kresimir-coko** [comment]
ci:test:sf

File: modules/apps/site/site-memberships-web/src/main/resources/META-INF/resources/js/UserManagementToolbarPropsTransformer.js [hunk 1]
```diff
@@ -56,6 +56,20 @@ export default function propsTransformer({portletNamespace, ...otherProps}) {
...
+					`${`${portletNamespace}roleId`}=${selectedItem.id}`,
+					itemData.viewRoleURL
+				);
+			},
+			selectEventName: `${portletNamespace}selectRole`,
+			title: Liferay.Language.get('select-role'),
+			url: itemData?.selectRolesURL,
+		});
+	};
+
```
Comment (victorg1991) [inline modules/apps/site/site-memberships-web/src/main/resources/META-INF/resources/js/UserManagementToolbarPropsTransformer.js:72]: Hey @kresimir-coko I think we may need to add this code also to UserGroupsManagementToolbarPropsTransformer.js

Comment (kresimir-coko) [reply modules/apps/site/site-memberships-web/src/main/resources/META-INF/resources/js/UserManagementToolbarPropsTransformer.js:72]: Hey @victorg1991, referencing our conversation from yesterday, I can't seem to find a commit where these functions were removed. PropsTransformer was added as a file and wasn't changed until now. I was looking at this commit https://github.com/liferay/liferay-portal/commit/d5285d8ccfa0b3b93b110c89fe7f33b57008af2d. 

I'm just wanted to confirm that we indeed do need to add `selectRoles` to `UserGroupsManagementToolbarPropsTransformer.js`, so if you can clarify it for me, that would be awesome 😄

Comment (victorg1991) [reply modules/apps/site/site-memberships-web/src/main/resources/META-INF/resources/js/UserManagementToolbarPropsTransformer.js:72]: Sure @kresimir-coko  :)

First of all, there is the same "roles" filter button in organization:

Memberships_-_Liferay

Also, checking the java code the roles dropdown items is added [here](https://github.com/liferay/liferay-portal/blob/a1f28cfad1cc5b41a200937a56187dc0ae7524b9/modules/apps/site/site-memberships-web/src/main/java/com/liferay/site/memberships/web/internal/display/context/UserGroupsManagementToolbarDisplayContext.java#L320-L347)

And lastly, this is the commit where the event handlers were removed: aae6ab06e5443f194d8fd63760629a9d60fcfbf2 
Note that in UserGroupsManagementToolbarDisplayContext there is also a selectRoles function that was removed :)

I hope this helps! 

Thanks!

**Comment by kresimir-coko** [comment]
@victorg1991 here we go, I've pushed a commit that adds the missing modal back in :) I've tested it works as intended.

**Comment by victorg1991** [comment]
LGTM :D Thanks!!

 @kresimir-coko  Those card were migrated in november https://github.com/liferay/liferay-portal/commit/0b62deed530a06aa26b1bd9ec5c278b97b73e3b9#diff-1f426ecbde39b4c4f695dee6a54a01abac79ea510fe1edf05a6810a4ee2e06e6 do we need to do something else with that?

**Comment by victorg1991** [comment]
ci:forward

**Comment by kresimir-coko** [comment]
@victorg1991 The card you are using in that file is extending from a `soy` one, as seen here https://github.com/liferay/liferay-portal/blob/master/modules/apps/site/site-memberships-web/src/main/java/com/liferay/site/memberships/web/internal/servlet/taglib/clay/SelectRoleVerticalCard.java#L17.

I believe that should not be the case, @julien you were the one to migrate these cards, got any input from the top of your head?

**Comment by kresimir-coko** [comment]
I reran `gradlew formatSource` on all affected modules, but nothing came up, I did get this error at the end of the task:
```
Dependency 'com.liferay:com.liferay.source.formatter:1.0.1098', 
published Thu Apr 01 01:54:42 CEST 2021, is older than the latest one available 
('com.liferay:com.liferay.source.formatter:1.0.1101', published Thu Apr 08 00:34:28 CEST 2021), but not yet expired
```

**Comment by victorg1991** [comment]
@kresimir-coko I think the sf problem not something related to this pr, there are several pulls failing. More info here: https://liferay.slack.com/archives/C19PWQ74J/p1617868824095800

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**